CRM/Utils/Check/Component/Env.php

```diff
@@ -963,14 +963,22 @@ public function checkPHPIntlExists() {
   public function checkWkHtmlToPDFPath() {
     $messages = [];
     $wkhtmltopdfPath = CRM_Core_Config::singleton()->wkhtmltopdfPath;
-    if (!empty($wkhtmltopdfPath) && !file_exists($wkhtmltopdfPath)) {
-      $messages[] = new CRM_Utils_Check_Message(
-        __FUNCTION__,
-        ts('CiviCRM is configured to use the wkhtmltopdf package to produce PDFs however it is missing, as such the system will fall back to using the DOMPDF package, this may mean that the output is different to what was expected. You should resolve this by either clearing the setting at Administer -> System Settings -> Miscellaneous or by getting your system administrator to install the wkhtmltopdf package'),
-        ts('Missing System Package: wkhtmltopdf'),
-        \Psr\Log\LogLevel::WARNING,
-        'fa-server'
-      );
+    if (!empty($wkhtmltopdfPath) {
+      // check and ensure that this leads to the wkhtmltopdf binary
+      // and it is a valid executable binary
+      // Only check the first space separated piece to allow for a value
+      // such as /usr/bin/xvfb-run -- wkhtmltopdf (CRM-13292)
+      $pieces = explode(' ', $fields['wkhtmltopdfPath']);
```

Comment:
Also what is `$fields` ?

Reply:
It was used in the file that I was copying from, you're right, I'm removing it.